webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-student.js

```diff
@@ -134,14 +134,14 @@ class SakaiRubricStudent extends RubricsElement {
         <div id="rubric-grading-or-preview" class="rubric-tab-content rubrics-visible">
           ${this.preview || this.forcePreview ? html`
           <sakai-rubric-criterion-preview
-            criteria="${JSON.stringify(this.rubric.criteria)}"
+            .criteria="${this.rubric.criteria}"
             .weighted=${this.rubric.weighted}
           ></sakai-rubric-criterion-preview>
           ` : html`
           <sakai-rubric-criterion-student
-            criteria="${JSON.stringify(this.rubric.criteria)}"
+            .criteria="${this.rubric.criteria}"
```

Comment:
You can just set the criteria directly as a property? No JSON stringify step? Nice :)

Reply:
Yes, it seems to work. But criteria is not an "attribute property", so it's not picked up as an string. That's how I explain it to myself.

Reply:
Fair enough. We don't need to pass it as a string on an attribute. Did you add attribute: false to the propery, too. You could probably do that.

Reply:
I did not add `attribute: false`, but I didn't remove them either. Those properties were without `attribute`. I'm not certain if this will break it honestly. But sure, we can add either `false` or a `"name"` later.

Reply:
I'm adding attribute: false everywhere where I know we're not reflecting back to the tag attribute. It's good documentation I think, and I think it improves the performance of the component - it doesn't set the tag attribute if you update the var in the component js.